### ISSUE TITLE
Project: Cache Insights

### ISSUE THREAD

**Issue by stephanie-anderson** [getsentry/team-sdks#62]
### Description

We are building a solution to monitor the performance of your cache (for ex. redis), showcasing things like cache hit/miss ratios, eviction rates and throughput. In order to support this, the SDK must collect extra data from the server. The following information is required:

_(To be filled out once we have aligned on what is/isn't possible from the notion doc below)_

### Timelines

Should be available by the end of FYQ1 (`May 15th, 2024`) and be supported by at least the following SDKs:
- Python SDK
- Node.js SDK

### Slack-Channel

`#discuss-performance`

### Notion Document(s)

- [Cache module requirements](https://www.notion.so/sentry/Cache-Module-f313edae8ad44b1bb47df9923b89cd5a?pvs=4)
- [SDK project representation in Notion](https://www.notion.so/sentry/SDKs-Starfish-Module-Caches-be04dfdb05d544d0a3b673d3925327c6)

### Stakeholder(s)

@alexjillard 
@zoesyc 
@DominikB2014
Benjamin Coe

### Team(s)

Web Backend, Web Frontend

## Tasks

```[tasklist]
### Web Backend
- [ ] https://github.com/getsentry/sentry-python/issues/2903
- [ ] https://github.com/getsentry/sentry-python/issues/2949
- [ ] https://github.com/getsentry/sentry-python/issues/2966
```

```[tasklist]
### Web Frontend
- [ ] https://github.com/getsentry/sentry-javascript/issues/11269
```

```[tasklist]
### Mobile
- [ ] tbd
```

**Comment by antonpirker** [comment]
FYI: I added the Python issue for adding a memcached integration that emits spans eligible for the new insights caches module. This integration is not part of the initial release and we will gather user feedback about the existing caches module before we start adding more instrumenations.

**Comment by stephanie-anderson** [comment]
Let's close this project issue for now and create separate issues in SDK repos, if any specific instrumentation should be added
